docs/sources/datasources/pyroscope/profilng-tracing-intro.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Profiling and tracing synergies
+menuTitle: Profiling and tracing
+description: Learning about how profiling and tracing work together.
+weight: 250
+aliases:
+  - ../introduction/traces-to-profiles/
+  - ../introduction/profile-tracing/
+keywords:
+  - pyroscope
+  - continuous profiling
+  - tracing
+---
+
+# Profiling and tracing synergies
```

Comment:
Personal nit, I don't like the word synergy, it feels too corporate.

Reply:
Agreed. I cringed when I used the word for similar reasons, but there wasn't a better word I could think of. The original page title was integration, but that was a loaded term. Open to suggestions!

Reply:
Yeah, it all seems a bit market-y to me rather than being documentation-y.

Interoperability is the word that came to mind but it's long and clunky. Perhaps a synonym of that? https://www.wordhippo.com/what-is/another-word-for/interoperability.html#:~:text=like%20this%20%E2%96%BC-,Noun,conformity

Reply:
I decided to change the page title to "How profiling and tracing work together" because that's what we're talking about. We can always revisit the page title, but I think that's clearer than any of the synonyms we've tried.